validations/src/validations.rs

```diff
@@ -299,6 +436,13 @@ pub fn validate_data_request_output(request: &DataRequestOutput) -> Result<(), T
         return Err(TransactionError::InsufficientWitnesses);
     }
 
+    let cond = request.min_consensus_percentage > 50 && request.min_consensus_percentage < 100;
+    if !cond {
```

Comment:
I'd simplify as:
```suggestion
    if request.min_consensus_percentage <= 50 || request.min_consensus_percentage >= 100  {
```

Reply:
You could also use range syntax although it's pretty rare to find it in the wild.

```suggestion
    if (51..=100).contains(&request.min_consensus_percentage) {
```